modules/operators-overview.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='operators-overview-{context}']
+= Operators in {product-title}
+
+In {product-title} version 4.0, all cluster functions are divided into a series
+of component Operators.
+
+Operators can manage other Operators within their component. This lends to a
+transparent model: a single Operator usually manages a single binary file.
+Operators also offer a more granual configuration experience. You configure each component by
+modifying its Operator instead of modifying a global configuration file.
+
+In version 4.0, separate processes and images and pods run to drive the kubernetes
+API and pod managers. The control plane services run as static pods so they can
+manage normal workloads or processes the same way that they manage disaster
+recovery. Aside from the core control plane components, other services run as 
+normal pods on the cluster, managed by regular Kubernetes constructs. Unlike in the past
+where the `kubelet` could be running as containerized or non-containerized, the `kubelet`
+always runs as a `systemd` process.
+
+
+[id='cluster-version-operator-{context}']
+== The cluster version Operator
+
+The cluster version Operator orchestrates all things.
+
+{product-title} 4.0 introduces several new components that support the cluster
+version Operator, including Cincinnati and Telemetry.
```

Comment:
Telemetry should be split out and part of an “observability” high level arch topic that then describes the major subsystems and what they do together (monitoring with Prometheus, alertmanager and grafana, logging, and telemetry).

Reply:
Do you either have a suggestion for a contact who can explain how these fit together or the outline for the user story?  I'm not familiar enough with how monitoring, alerting, grafana, logging, and telemetry fit together to say what that story is, and all of our assemblies need to have one.